.github/workflows/ruby.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
```suggestion
    branches: "*"
```

Reply:
I think we're not seeing any jobs run because the workflow is configured to only run for the master branch.

Reply:
I think this field is actually for which branch the PR is targeting to merge into, so this should have been fine. But I still think you'd probably want to run tests on a PR merging into a different branch anyway.